src/App.tsx

```diff
@@ -1,57 +1,38 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
+import 'bulma';
 import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<string | null>(null);
 
   return (
     <main className="section">
-      <h1 className="title">
-        {`${name} (${born} = ${died})`}
-      </h1>
-
-      <div className="dropdown is-active">
-        <div className="dropdown-trigger">
-          <input
-            type="text"
-            placeholder="Enter a part of the name"
-            className="input"
-          />
-        </div>
-
-        <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Petronella de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Hercke</p>
-            </div>
-          </div>
-        </div>
-      </div>
+      {selectedPerson && (
+        (() => {
+          const foundPerson = peopleFromServer.find(
+            (person) => person.name === selectedPerson,
+          );
+
+          if (foundPerson) {
+            return (
+              <h1 className="title" key={selectedPerson}>
+                {`${foundPerson.name} (${foundPerson.born} - ${foundPerson.died})`}
+              </h1>
+            );
+          }
+
+          return null;
+        })()
```

Comment:
I don't like this anonymous function to be honest. Can you please make it just like a regular `jsx`?